README.md

```diff
@@ -126,6 +126,7 @@ smartive_handlebars:
 There are two caching services / strategies available per default:
 - `smartive_handlebars.cache.disk`: Uses [Handlebars\Cache\Disk](https://github.com/XaminProject/handlebars.php/blob/master/src/Handlebars/Cache/Disk.php) to read/write file cache in Symfony's cache directory
 - `smartive_handlebars.cache.apc`: Uses [Handlebars\Cache\APC](https://github.com/XaminProject/handlebars.php/blob/master/src/Handlebars/Cache/APC.php) to read/write cache objects using [APC](http://php.net/manual/en/book.apc.php)
+- `smartive_handlebars.cache.redis`: Uses the [RedisBundle](https://github.com/snc/SncRedisBundle) to read/write cache objects using a [Redis Server](http://redis.io/). Per default, the `snc_redis.default` client is being used. To overwrite this, add your own service definition.
```

Comment:
what about "... add your own service definition or do a pull request on this bundle to make the redis client configurable."? it would not be that hard, but i agree we don't need it ourselves so we should not bother.